yogstation/code/modules/clothing/donator.dm

```diff
@@ -184,3 +184,9 @@
 	item_state = "axolotl"
 	attack_verb = list("nibbles", "splats")
 	squeak_override = list('sound/effects/axolotl.ogg' = 1)
+/obj/item/toy/plush/gslimeplushie
+	name = "galactic slime"
+	desc = "usually it would be impossible to get something like this...But do to your current situation."
```

Comment:
```suggestion
	desc = "Usually it would be impossible to get something like this... But due to your current situation."
```
wierd way to end a sentence